files/packages/__name__/.eslintrc.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
```

Comment:
no typescript support? :( 

I have this setup here: https://github.com/NullVoxPopuli/eslint-configs/blob/main/configs/ember.js#L180-L181

Reply:
But if you want. to support TS, then you would also have to have a different rollup config. 

I did this for https://github.com/simonihmig/tracked-redux, for which your previous work inspired me (e.g. I didn't know about `rollup-plugin-ts`, which worked really nicely), so thanks for that! 

For me TS support was out of scope for the blueprint here, for now, as the original one also doesn't have it. [RFC776](https://emberjs.github.io/rfcs/0776-typescript-blueprints.html) will add a `--typescript` flag, so we should eventually do the same here IMO.

Reply:
> But if you want. to support TS, then you would also have to have a different rollup config.

ya, this is being tracked here: https://github.com/embroider-build/embroider/pull/1099
(as well as all the blockers before things are ready for production use (many blockers are also blockers for JS usage))

I have sample ts/js configs here in the mean time that I'm testing: https://github.com/NullVoxPopuli/ember-addon-migrator/tree/main/src